### ISSUE TITLE
posh-ssh with RMM tools - Requested Registry access is not allowed

### ISSUE THREAD

**Issue by markwiater** [darkoperator/Posh-SSH#136]
I'm trying to run a script using posh-ssh within a remote monitoring and management tool. Connectwise Automate in this case.

The script runs properly from a normal command line with a logged on user. However Connectwise most commonly does it's work as the localsystem account. I'm getting an error regarding registry access.

I'm wondering, if the registry is only used for is storing keys (that's all I could find it doing), if it might not be supportable to use a portable csv file instead?

**Comment by darkoperator** [comment]
Yes registry is used for storing keys, no it does not support key storage or validation via CSV. In a future release I can make it so when one disables key validations it does not try to open the registry key.

**Comment by markwiater** [comment]
Thanks Carlos. That sounds great.   Any ideas when that future release might be available? 

I'd also like to respectfully request consideration of an alternate method (different from the registry) to validate keys at some point way in the future. 

Thanks again

**Comment by darkoperator** [comment]
No clue on when I Would have version 2.0 out, have so many projects I only work on it once or twice a year on it a bit unless a major bug comes out

**Comment by darkoperator** [comment]
will try to push version 2.0 after I'm finished with DEFCON

**Comment by darkoperator** [comment]
Yes it did, just use -Force

**Comment by markwiater** [comment]
Thank you!
